parsers/ENTSOE.py

```diff
@@ -164,11 +165,10 @@
 }
 # Some zone_keys are part of bidding zone domains for price data
 ENTSOE_PRICE_DOMAIN_OVERRIDE = {
-    'AT': ENTSOE_DOMAIN_MAPPINGS['DE-AT-LU'],
```

Comment:
AT by default gives the 15min prices as @alixunderplatz noted. We should probably set the right code for the AT bidding zone I think? Or did ENTSOE update such that AT now shows the correct prices?

Reply:
@corradio we can use bidding zone Austria, it has the correct prices now as you assumed.

![image](https://user-images.githubusercontent.com/25743609/46368876-a2d8d400-c681-11e8-9444-dc5a6d673ca9.png)

Reply:
'CTY|10YAT-APG------L' should be the correct ENTSO-E code for the bidding zone

Reply:
@lorrieq I just saw your sample output, everything looks fine there pricewise! :)